dask_expr/expr.py

```diff
@@ -208,39 +209,77 @@ def _layer(self) -> dict:
 
         return {(self._name, i): self._task(i) for i in range(self.npartitions)}
 
-    def simplify(self):
+    def simplify(
+        self,
+        phases: list
+        | tuple = (
+            "general",
+            (
+                "general",
+                "lower",
+            ),
+        ),
+    ):
         """Simplify expression
 
-        This leverages the ``._simplify_down`` method defined on each class
+        This leverages the ``_simplify_up`` and ``_simplify_down``
+        methods defined on each class (via ``_simplify``)
+
+        Parameters
+        ----------
+        phases: Optional
+            Specific optimization phases to perform. Default is
+            ``(("general",), ("general", "lower"))``.
 
         Returns
         -------
         expr:
             output expression
-        changed:
-            whether or not any change occured
         """
+
+        # Loop over all phases
+        expr = self
+        for phase in phases:
+            allow_group = (phase,) if isinstance(phase, str) else phase
+            if not isinstance(allow_group, tuple):
+                raise TypeError("Each element of `phases` must be str or tuple")
```

Comment:
I would be more strict here instead of introducing some magic (from a user pov). Raise if the phase isn't a tuple